pipelines/MetricsRuntime.yaml

```diff
@@ -24,54 +24,38 @@ tasks:
       connections.metric: CalibrateTime
       connections.labelName: calibrate
       target: calibrate.run
-  timing_imageDifference:
+  timing_subtractImages:
     class: lsst.verify.tasks.commonMetrics.TimingMetricTask
     config:
-      connections.package: pipe_tasks
+      connections.package: ip_diffim
       connections.metric: ImageDifferenceTime
```

Comment:
Renaming `pipe_tasks.ImageDifferenceTime` to `ip_diffim.ImageDifferenceTime` needs to be done in `verify_metrics`.

If you're renaming the metric anyway, would it be better to call it `SubtractImagesTime`? Our SQuaSH graphs will need to be modified either way (and that may be tricky, since the namespace is fundamental to how metrics are organized in the online database 😬).

Reply:
I'll make the `verify_metrics` PR. What else is needed for these metrics to be picked up by SQuaSH?

Reply:
That should be it.